### PR TITLE
Oy2 9929 - PPC-AD

### PR DESCRIPTION
**Pull request by jdinh8124**
## Purpose
oy2 9929 PPC-AD

_Describe the problem or feature in addition to a link to the issues._

#### Linked Issues to Close
https://qmacbis.atlassian.net/jira/software/c/projects/OY2/boards/208?modal=detail&selectedIssue=OY2-9929

#### Pull Request Creator Checklist

- [x] This PR has an associated issue or issues.
- [x] The associated issue(s) are linked above.
- [x] This PR meets all acceptance criteria for those issues.
- [x] This PR and linked issue(s) are adequately documented
- [x] This PR and linked issues(s) are a complete description of the changeset; an individual or team should be able to understand the issue(s) and changes by reading through this PR and it's links, with no further interaction.
- [x] Someone has been assigned this PR.
- [x] At least one person has been marked as reviewer on this PR.

#### Pull Request Reviewer/Assignee Checklist

- [ ] This PR has an associated issue or issues.
- [ ] The associated issue(s) are linked above.
- [ ] This PR meets all acceptance criteria for those issues.
- [ ] This PR and linked issue(s) are adequately documented
- [ ] This PR and linked issues(s) are a complete description of the changeset; an individual or team should be able to understand the issue(s) and changes by reading through this PR and it's links, with no further interaction.

### CODE REVIEW

File: services/ui-src/src/measures/globalValidations/dataDrivenTools.ts [hunk 1]
```diff
@@ -91,15 +91,21 @@ export const getDeviationNDRArray = (
...
-    if (ageGroups) {
-      for (const key of Object.keys(objectToSearch).filter(
-        (prop) => prop !== DC.SELECTED_OPTIONS
-      )) {
-        deviationArray.push(data[option][key as Types.DeviationKeys]);
-      }
-    } else {
-      if (data) {
-        deviationArray = Object.values(data);
+    if (objectToSearch) {
```
Comment (benjaminpaige) [inline services/ui-src/src/measures/globalValidations/dataDrivenTools.ts:94]: 👍

**Review by benjaminpaige — COMMENTED** [review]
love the use of the constants : )

**Comment by abelzer212** [comment]
The Second P in Postpartum shouldn't be capitilized
Screen Shot 2022-03-17 at 1 52 58 PM

File: services/ui-src/src/measures/globalValidations/validationsLib.tsx [hunk 2]
```diff
@@ -366,9 +366,9 @@ export const validateAtLeastOneNDRInDeviationOfMeasureSpec = (
     if (ndrCount > 0) {
       const atLeastOneDevNDR = deviationArray.some((deviationNDR: any) => {
         if (
-          deviationNDR.denominator &&
-          deviationNDR.numerator &&
-          deviationNDR.other
+          deviationNDR?.denominator &&
+          deviationNDR?.numerator &&
+          deviationNDR?.other
```
Comment (benjaminpaige) [inline services/ui-src/src/measures/globalValidations/validationsLib.tsx:371]: : )

**Review by benjaminpaige — APPROVED** [review]
if we could use the dc. that owuld be great. otherwise its great!!

**Pull request merged**